introduction.md

```diff
@@ -1,2 +1,6 @@
 hey..this is my first file on git.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?